CONTRIBUTING.md

```diff
@@ -245,3 +245,34 @@ npm run update_prebuilt
 
 That last step is easy to forget, so try to make it a habit whenever you're
 prepping a PR that includes frontend work.
+
+### MacOS
+
+Install Xcode and Homebrew Mac. Once that is done: 
+
+```bash
+#Install dependent packages
```

Comment:
I'd just call these "dependencies". In this case, I'd say OpenHTF is on the "dependent" side of the relation.